READMEs/marketplace-flow.md

```diff
@@ -5,20 +5,20 @@ SPDX-License-Identifier: Apache-2.0
 
 # Quickstart: Marketplace Flow
 
-This quickstart describes a batteries-included flow including using off-chain services for metadata (Aquarius) and consuming datasets (Provider).
+This quickstart describes a batteries-included flow including using off-chain services for metadata (Aquarius).
 
 For pool creation, it is used as base token, OCEAN token.
 The base token can be changed into something else, such as USDC, DAI etc., but
 it will require an extra fee.
 
-It focuses on Alice's experience as a publisher, and Bob's experience as a buyer & consumer.
+It focuses on Alice's experience as a publisher, and Bob's experience as a buyer.
 
 Here are the steps:
 
 1.  Setup
 2.  Alice publishes data asset
 3.  Market displays the asset for sale
-4.  Bob buys data asset, and downloads it
+4.  Bob buys data asset
 
 Let's go through each step.
```

Comment:
In section "2. Publish Data NFT & Datatoken", this has big DRY violations.

To fix, remove _all_ its content including the reference to datatokens-flow.md, and replace it with a reference to consume-flow.md section "2. Publish Data NFT & Datatoken".

Reply:
this would require a new dependency to setup in the automated readme tests. I.e the marketplace flow would depend on two previous file runs, and they are not even common (running an end-to-end readme as a dependency to our currently-testing readme). There are two solutions to this:

1. keep the DRY violation for publishing for now, since it will take a lot of time to setup the automation here, maybe even impossible in this current form of the readmes.
2. create a separate readme for publishing, which we refer as part of both marketplace flow and consume flow.

I addressed all the other changes.

Reply:
Thanks for the options.

Given the size of this DRY violation, (1) is not a great choice.

Therefore let's do (2). Could call it "publish with metadata" or something.